lib/ejs.js

```diff
@@ -86,15 +86,16 @@ exports.localsName = _DEFAULT_LOCALS_NAME;
  *
  * @param {String} name     specified path
  * @param {String} filename parent file path
+ * @param {bool} filename is dir?
```

Comment:
In JSDoc syntax, boolean is `{boolean}` (or `{Boolean}`)

Also change `filename` to `isdir`, and `is dir?` to <code>if \`filename\` is the directory rather than the file</code>.

Also vertically align the type, the parameter name, and the description.